include/tesla.hpp

```diff
@@ -3484,6 +3653,7 @@ namespace tsl::cfg {
     u16 FramebufferWidth  = 0;
     u16 FramebufferHeight = 0;
     u64 launchCombo = KEY_L | KEY_DDOWN | KEY_RSTICK;
+    u64 captureCombo = KEY_PLUS | KEY_MINUS;
```

Comment:
Since you added global variables for this, mind making in a config option in the config.ini as well?
Maybe have a `screenshot_combo_enabled` and `screenshot_combo` option there